connectors/src/lib/nango_helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { redisClient } from "@connectors/lib/redis";
+import { NangoConnectionId } from "@connectors/types/nango_connection_id";
+
+import { nango_client } from "./nango_client";
+
+const NANGO_ACCESS_TOKEN_TTL_SECONDS = 60 * 5; // 5 minutes
+
+export async function getAccessTokenFromNango({
+  connectionId,
+  integrationId,
+  useCache = false,
+}: {
+  connectionId: NangoConnectionId;
+  integrationId: string;
+  useCache?: boolean;
+}) {
+  const cacheKey = `nango_access_token:${integrationId}/${connectionId}`;
+  const redis = await redisClient();
```

Comment:
I might be missing something but aren't we supposed to quit the client?

Reply:
We are! And in a try / catch / finally, otherwise we won't quite on error.
That makes me think that we should provide a helper like sequelize.transaction:
```
redis.connection((redisClient) => {
redisClient.set()....

})
```

Reply:
I am checking the number of redis conenctions right now in production.

Reply:
Ok I can fix

Reply:
I'm def missing something since I don't see the `quit` neither do I see the finally?

Reply:
Ah ok gotcha misunderstood your comment so we are not. And we should :+1: 

@fontanierh yours?

Reply:
@spolu I am saying  "...but aren't we supposed to quit/disconnect the client?" -> "... We are! And in a try / catch / finally, otherwise we won't quite on error."

Reply:
The number of connections to the redis server spiked from ~25 avg up to a 46k peak. Need to fix ASAP.

Reply:
yes

Reply:
<img width="1455" alt="Screen Shot 2023-10-25 at 13 09 06" src="https://github.com/dust-tt/dust/assets/358965/6988f9ab-59bf-4c29-ad29-eba19410d02e">